tools/halide_image.h

```diff
@@ -554,38 +554,38 @@ struct for_each_element_helpers {
      * resolution. The decltype is to make this version impossible if
      * the function is not callable with this many args. */
     template<typename ...Args>
-    static auto for_each_element_variadic(int, int d, Fn f, const buffer_t &buf, const Args &... args)
```

Comment:
I can back these out if you like -- was just trying to pre-emptively get everything, but may have overreached.